save-core/src/commonNonJsMain/kotlin/com/saveourtool/save/core/utils/TestConfigUtils.kt

```diff
@@ -20,6 +20,16 @@ fun TestConfig.processInPlace() = processInPlace {
     createPluginConfigListFromToml(it.location, fs)
 }
 
+/**
+ * Process current and all parents of [this] config, reading individual plugin configurations from TOML file.
+ *
+ * @return [this] config with all descendants being evaluated (`pluginConfigs` are filled with data and merged with parents)
+ */
+fun TestConfig.processWithParentsInPlace(): TestConfig {
+    parentConfig?.processInPlace()
```

Comment:
> Process current and **all** parents

Shouldn't it like this be in aim to process all parents?

```suggestion
    parentConfig?.processWithParentsInPlace()
```

Reply:
yes, good catch

Reply:
fixed + added a test for it